src/components/security_manager/test/ssl_certificate_handshake_test.cc

```diff
@@ -335,6 +342,9 @@ class SSLHandshakeTest : public testing::Test {
   std::string client_certificate_;
   std::string client_ciphers_list_;
   std::string client_ca_certificate_path_;
+
+  const std::vector<int> forced_protected_service;
+  const std::vector<int> forced_unprotected_service;
```

Comment:
@ValeriiMalkov same as above

Reply:
@AByzhynar fixed at the  commit 0eee356873e4c810e70e48657dc1f4dc1a8467fe